addons/skin.estuary/xml/Constants_1920.xml

```diff
@@ -7,5 +7,5 @@
 	<constant name="tvrecordings_width">1060</constant>
 	<constant name="eventloglist_width">1430</constant>
 	<constant name="playlisteditorlist_width">770</constant>
```

Comment:
these files dont exist anymore in my wip branch https://github.com/phil65/skin.estuary/commits/master